modules/gapi/misc/python/pyopencv_gapi.hpp

```diff
@@ -380,10 +380,10 @@ static cv::detail::OpaqueRef extract_opaque_ref(PyObject* from, cv::detail::Opaq
         HANDLE_CASE(DOUBLE,  double);
         HANDLE_CASE(FLOAT,   float);
         HANDLE_CASE(STRING,  std::string);
-        HANDLE_CASE(POINT,   cv::Point);
-        HANDLE_CASE(POINT2F, cv::Point2f);
-        HANDLE_CASE(SIZE,    cv::Size);
-        HANDLE_CASE(RECT,    cv::Rect);
+        HANDLE_CASE(POINT,   cv::Point  , -1, -1);
+        HANDLE_CASE(POINT2F, cv::Point2f, -1.f, -1.f);
+        HANDLE_CASE(SIZE,    cv::Size   , -1, -1);
+        HANDLE_CASE(RECT,    cv::Rect   , -1, -1, -1, -1);
```

Comment:
I believe we shouldn't do this, do these structures lack their default constructor initialization?

Reply:
cc: @alalek @mshabunin

Reply:
Right, this is not needed as these classes has own default constructors.

E.g., here: https://github.com/opencv/opencv/blame/4.5.2/modules/core/include/opencv2/core/types.hpp#L1159-L1161